docs/advanced/csidriver.md

```diff
@@ -51,25 +59,200 @@ With the kubeconfig of the Harvester cluster, you can install the third-party CS
+
+![created-image-with-nfs-csi](/img/v1.5/advanced/created-image-with-nfs-csi.png) 
+
+### Virtual Machine Creation
+
+Your virtual machines can use root and data volumes in external storage.
+
+When [creating a virtual machine](../vm/create-vm.md) using the Harvester UI (**Virtual Machine > Create**), you must perform the following actions on the **Volumes** tab:
+
+- Select a virtual machine image stored in the external storage solution, and then configure the required settings.
+- Add a data volume.
+
+![various-volumes-for-vm-creating](/img/v1.5/advanced/various-volumes-for-vm-creating.png)
+
+In the following example, the root volume is created using NFS, and the data volume is created using the Longhorn V2 Data Engine.
+
+![various-volumes-for-vm-created](/img/v1.5/advanced/various-volumes-for-vm-created.png)
+
+### Volume Creation
+
+You can create volumes in your external storage solution.
+
+When [creating a volume](../volume/create-volume.md) using the Harvester UI (**Volumes > Create**), you must perform the following actions:
+
+- **Storage Class**: Select the target StorageClass, e.g. **nfs-csi**.
+- **Volume Mode**: Select the corresponding volume mode, e.g. **Filesystem** for **nfs-csi**.
+
+![create-fs-volume](/img/v1.5/advanced/create-fs-volume.png)
+
+## Advanced Topics
+
+### Storage Profiles
+
+You can now use the CDI API to create custom [storage profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) that simplify definition of data volumes. Storage profiles allow multiple data volumes to share the same provisioner settings.
+
+The following is an example of an LVM storage profile:
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: lvm-node-1-striped
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+status:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+  cloneStrategy: snapshot
+  dataImportCronSourceFormat: pvc
+  provisioner: lvm.driver.harvesterhci.io
+  snapshotClass: lvm-snapshot
+  storageClass: lvm-node-1-striped
+```
+
+For more information, see [Storage Profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) in the CDI documentation.
+
+You can define the above fields to override the default configuration showing on the status.
+
+### Limitations
+
+- Backup support is currently limited to Longhorn V1 Data Engine volumes. Harvester is unable to create backups of volumes in external storage. 
+
+- There is a limitation in the CDI which prevents Harvester from converting attached PVCs to virtual machine images. Before exporting a volume in external storage, ensure that the PVC is not attached to workloads. This prevents the resulting image from getting stuck in the *Exporting* state.
+
+![convert-pvc-to-image-stuck](/img/v1.5/advanced/convert-pvc-to-image-stuck.png)
+
+### How to deploy the NFS CSI driver
```

Comment:
Why do we need to include this in our docs? It feels a bit out of place. If any of the upstream csi nfs instructions changed, our documentation will become stale. Also, the provisioner at https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner seems very old. If we are going to recommend it in our docs, we need to make sure their images are safe and updated.

Alternately, what do you of converting this content into a blog post or recording or kubecon talk proposal where we showcase how to use nfs csi driver and storage profile in 1.5? I can help with this (especially, kubecon proposal :wink:), if needed.

Reply:
If we really want to include this in our docs, maybe consider just add a href to https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/example/README.md and call it done?

Reply:
yeah, i prefer not to include it in our docs. many of its closed issues indicated that the project is no longer maintained. any reasons why we don't want to link to the csi nfs official docs at https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/example/README.md?

Reply:
OK, let's remove!